### ISSUE TITLE
NavCollapse has fixed height on first collapse

### ISSUE THREAD

**Issue by acramatte** [gwtbootstrap/gwt-bootstrap#455]
There is an issue when using a ResponsiveNavbar containing Dropdowns.

When the screen is too small, there is that NavbarButton that appears to collapse hidden content and when you collapse the content <b>the first time</b>, the NavCollapse that holds the content will have a fixed height size. Therefore when clicking on a dropdown, the size won't adapt and some of the content will be hidden.

If we fold the content and collapse it again, it works as it should.

This issue can be reproduced with the Navbar that's on the [demo page](http://gwtbootstrap.github.io/#component:navigation)

Basically the issue is the same as the one described [here](http://stackoverflow.com/questions/14735747/bootstrap-responsive-collapsed-submenu-has-fixed-height-on-first-collapse) and the fix for it is the same.
